src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,94 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final double MAGNIFICATION_FACTOR = 1.5;
+    private static int defaultCapacity = 10;
```

Comment:
it should be constant 
if you need to know current array length, just use `elements.length`